samples/matter/lock/Kconfig

```diff
@@ -32,6 +55,15 @@ config LOCK_ENABLE_DEBUG
 
 if LOCK_ENABLE_DEBUG
 
+config LOCK_PRINT_STORAGE_STATUS
```

Comment:
Is it used anywhere? I can't find it.

Reply:
It is used in the `credentials_storage.cpp` please check one more time. Previously it was used but never defined in Kconfig. It is a fix for that.